e2e/02-login-flow.e2e.spec.ts

```diff
@@ -49,8 +49,9 @@ describe("Login Flow", () => {
       )
       await tokenInput.waitForDisplayed({ timeout })
       await tokenInput.click()
-
-      await tokenInput.sendKeys(mobileUserToken?.split(""))
+      console.log(mobileUserToken)
```

Comment:
is log intentional?

Reply:
eslint rules preventing it has been removed

Reply:
No not intentional - will remove